src/cli/scaffold/rust/Cargo.toml.template

```diff
@@ -0,0 +1,14 @@
+[package]
+  name = "code"
```

Comment:
Name can be something besides "code" that was just the auto generated one, based on it being added within a folder named code

Reply:
Something more clear as a placeholder, to do with zomes